Eatery/Controllers/Eateries/CampusMenuViewController.swift

```diff
@@ -405,3 +405,13 @@ extension CampusMenuViewController: TabbedPageViewControllerDelegate {
     }
 
 }
+
+extension CampusMenuViewController: Reloadable {
+    func reload() {
+        childViewControllers.forEach { controller in
+            if let controller = controller as? Reloadable {
+                controller.reload()
+            }
+        }
```

Comment:
Use a for loop here. It's simpler and more concise. Also, I think you can do [this cool thing](https://stackoverflow.com/a/55990348):

```suggestion
        for case let controller as Reloadable in childViewControllers {
                controller.reload()
            }
        }
```

You can do something similar in a bunch of places.